### ISSUE TITLE
Create a basic mobx example

### ISSUE THREAD

**Issue by KyleAMathews** [electric-sql/electric#1477]
It should be similar to the other basic examples.

**Comment by msfstef** [comment]
I'll pick this up

**Comment by marbemac** [comment]
Just a heads up, from the mobx docs - `EcmaScript private fields (#field) are not supported by make(Auto)Observable`. `ShapeStream` and `Shape` use a pattern where they declare ecmascript private fields, and expose them via getters. Any particular reason to avoid using boring old typescript private property instead?

**Comment by KyleAMathews** [comment]
Hmm how are you doing the integration? This was my little mockup last year

https://github.com/electric-sql/archived-electric-next/pull/22#issuecomment-2229346877

**Comment by marbemac** [comment]
Yeah I ended up using subscribe on the `ShapeStream`. However if #fields were avoided, we could call `makeObservable` on ShapeStream or Shape instances, and then work off of reactive `offset`, `rows`, etc.
